FC-0068 docs: How-to cleaning by edunext #10 (#715)
openedx/docs.openedx.org: source/educators/how-tos/advanced_features/lti_determine_content_address.rst

The identifier for your course can be in one of these formats.

* ``{key type}:{org}+{course}+{run}``, for example,
  ``course-v1:OpenedX+DemoX+2024``

> * ``{org}/{course}/{run}``, for example, ``OpenedX/DemoX/2024``

Reviewer: ```suggestion
* ``{org}/{course}/{run}`` (for courses created prior to 2015), for example, ``OpenedX/DemoX/2014``
```

Courses created since Fall 2014 typically have an ID that uses the first
format, while older courses have IDs that use the second format.

To find the course ID for your course, follow these steps.

#. In your Open edX LMS, open your course.

#. In the URL shown by your browser, find the course ID.

For example, you open the "Open edX Demo Course" course to the **Course**
page for the course. The URL for the **Course** page is

> ``https://apps.sandbox.openedx.org/learning/course/course-v1:OpenedX+01-2024+2024-1/home``. From

Reviewer: ```suggestion
``https://training.openedx.io/courses/course-v1:OpenedX+DemoX+Demo_Course/about``. From
```

> the URL, you determine that the course ID is ``course-v1:OpenedX+01-2024``.

Reviewer: ```suggestion
the URL, you determine that the course ID is ``course-v1:OpenedX+DemoX+Demo_Course``.
```

be in one of these formats.

* ``{key type}:{org}+{course}+{run}+type@{type}+block@{display name}``, for

>   example, ``block-v1:OpenedX+DemoX-2024+2024-1+type@sequential+block@basic_questions``

Reviewer: ```suggestion
  example, ``block-v1:OpenedX+DemoX+Demo_Course+type@sequential+block@basic_questions``
```

   :widths: 45 45
   :header-rows: 1

>    * - Open EdX Studio

Reviewer: ```suggestion
   * - Studio
```

#. To find the usage ID for a component, find the **location**.

>    For example, ``location = block-v1:demo+demox+01+type@html+block@054cef851ecc415e969cd82c06a3307b``

Reviewer: ```suggestion
   For example, ``location = block-v1:OpenedX+DemoX+Demo_Course+type@html+block@054cef851ecc415e969cd82c06a3307b``
```

#. To find the usage ID for a unit, scroll down to find the **parent**.

>    For example, ``parent  demo+demox+01+type@vertical+block@7be7c1ea72f94d08b8bca998aa81f898``

Reviewer: ```suggestion
   For example, ``parent  block-v1:OpenedX+DemoX+Demo_Course+type@vertical+block@7be7c1ea72f94d08b8bca998aa81f898``
```

#. Use your browser's Find feature to locate the term ``data-usage-id``. This
   attribute contains the usage ID.

>    ..This step needs review because is not working in the last versions of Open edX.

Reviewer: ```suggestion
   ..note:: This step needs review because is not working in the last versions of the Open edX platform.
```

     be sure that you specify the content that you want.

For example, you want to link to a subsection in the Open edX Demo course. You open
the course, go to the problem, and then right click to view the page source.
When you search for ``data-usage-id``, the first match is

> ``block-v1:OpenedX+Demo+Demo_Course+type@sequential+block@basic_questions``. You

Reviewer: ```suggestion
``block-v1:OpenedX+DemoX+Demo_Course+type@sequential+block@basic_questions``. You
```
